### PR TITLE
Optimize log print

### PR DESCRIPTION
**Pull request by lightshine001**

### CODE REVIEW

**Review by laizy — DISMISSED** [review]

File: smartcontract/service/neovm/neovm_service.go [hunk 1]
```diff
@@ -255,7 +255,7 @@ func (this *NeoVmService) getContract(address []byte) ([]byte, error) {
 	if err != nil {
 		return nil, errors.NewErr("[getContract] Get contract context error!")
 	}
-	log.Infof("invoke contract address:%x", scommon.ToArrayReverse(address))
+	log.Debugf("invoke contract address:%x", scommon.ToArrayReverse(address))
```
Comment (laizy) [inline smartcontract/service/neovm/neovm_service.go:258]: use address.ToHexString

File: txnpool/proc/txnpool_server.go [hunk 2]
```diff
@@ -172,6 +170,7 @@ func (s *TXPoolServer) init(num uint8, preExec bool) {
 	}
 
 	s.gasPrice = getGasPriceConfig()
+	log.Infof("tx pool: initial enforced gas price %d", s.gasPrice)
```
Comment (Arbio5zt) [inline txnpool/proc/txnpool_server.go]: I think this message be " The current local gasprice is :xxxx " would be better.

File: smartcontract/service/neovm/neovm_service.go (hunk 1 above)
Comment (lightshine001) [reply smartcontract/service/neovm/neovm_service.go:258]: []byte has no method ToHexString

File: txnpool/proc/txnpool_server.go (hunk 2 above)
Comment (lightshine001) [reply txnpool/proc/txnpool_server.go]: Updated

**Review by laizy — APPROVED** [review]

**Pull request merged**